controllers/tasksRequests.js

```diff
@@ -0,0 +1,101 @@
+const { INTERNAL_SERVER_ERROR, SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+const taskRequestsModel = require("../models/taskRequests");
+
+const fetchTaskRequests = async (_, res) => {
+  try {
+    const data = await taskRequestsModel.fetchTaskRequests();
+
+    return res.status(200).json({
+      message: "Task requests returned successfully",
+      data,
+    });
+  } catch (err) {
+    logger.error("Error while fetching task requests", err);
+    return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
+  }
+};
+
+const addOrUpdate = async (req, res) => {
+  try {
+    const { taskId, userId } = req.body;
+
+    if (!taskId) {
+      return res.boom.badRequest("taskId not provided");
+    }
+    if (!userId) {
+      return res.boom.badRequest("userId not provided");
+    }
+
+    const response = await taskRequestsModel.addOrUpdate(taskId, userId);
+    if (response.userDoesNotExist) {
+      return res.boom.conflict("User does not exist");
+    }
+    if (response.userStatusDoesNotExists) {
+      return res.boom.conflict("User status does not exists");
+    }
+    if (response.requestorExists) {
```

Comment:
we discussed about changing it right? can we change this to `alreadyRequested`, `requestorExists` suggests me that another requestor is already exiting for this